### PR TITLE
Improve testing docs

### LINKED ISSUE #51: Document testing
The tests require a server to run but they also require a certificate. This is not documented at the moment.

And perhaps add support for using a PEM for the certificate instead of DER.

### PR DESCRIPTION
**Pull request by dbrgn**
Fixes #51.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,8 +17,42 @@ everything feasible.
...
 ## Testing
 
-**Note:** The tests currently expect a [SaltyRTC Server][server] instance to
-run on `localhost:6699`.
+### Setup
+
+The integration tests currently expect a [SaltyRTC Server][server] instance to
+run on `localhost:8765`.
+
+First, create a test certificate for localhost, valid for 5 years.
```
Comment (lgrahl) [inline README.md]: I don't think the validity matters here. :)

File: .circleci/config.yml [hunk 2]
```diff
@@ -24,7 +24,7 @@ jobs:
           command: /saltyrtc/certs/generate-cert.sh
       - run:
           name: Copy certificate
-          command: cp /saltyrtc/certs/saltyrtc.der .
+          command: cp /saltyrtc/certs/saltyrtc.crt .
```
Comment (lgrahl) [inline .circleci/config.yml:27]: Don't have a strong opinion about it but I'd usually use the `.pem` ending.

Comment (dbrgn) [reply .circleci/config.yml:27]: True, but `crt` is more consistent with the other clients' test instructions.

File: README.md (hunk 1 above)
Comment (dbrgn) [reply README.md]: Removed

**Pull request merged**